### ISSUE TITLE
Is qfind stable? Can it be used for IO-500? 

### ISSUE THREAD

**Issue by JulianKunkel** [carlilek/qfind#1]
Thanks for your help :-)

**Comment by carlilek** [comment]
I haven't visited this one for a couple years, so I am not sure if it even works with the current software release! I will have a look and see if I can figure that out...

**Comment by carlilek** [comment]
Yeah, looks like I'd have to do quite a few generalizations in this code to make it... well, general use.
